### PR TITLE
docker: move the docker image capture logic in ticker

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

If DOCKER_BUILD_VERBOSE=true is set, do not capture the docker image
output which is useful for debuging.

## Testing

* make -C securedrop dev
* DOCKER_BUILD_VERBOSE=true make -C securedrop dev
* make -C admin test
* DOCKER_BUILD_VERBOSE=true make -C admin test
* make ci-lint
* DOCKER_BUILD_VERBOSE=true make ci-lint

## Deployment

N/A

### CODE REVIEW

File: devops/scripts/ticker [hunk 1]
```diff
@@ -12,13 +12,24 @@ function display_dots() {
...
+    out="$(mktemp)"
     display_dots &
     pid="$!"
-    if "$@" ; then
-        status=0
+    status=0
+    if test -n "${DOCKER_BUILD_VERBOSE:-}" ; then
+        "$@" || status=1
+    else
+        echo "Run with DOCKER_BUILD_VERBOSE=true for more information"
```
Comment (redshiftzero) [inline devops/scripts/ticker]: We should also add this to the documentation, as I expect new contributors might miss this in the output but might see it in a [tip](http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions) box in the documentation, e.g. "Tip: If you're having trouble with your build, try..."

**Comment by dachary** [comment]
@redshiftzero good idea for the documentation, thanks for suggesting it. Added to the [SQUASH: docker: move the docker image capture logic in ticker](https://github.com/freedomofpress/securedrop/pull/3066/commits/02855649ae782494d16514f85d42311742f405f4) commit.

**Comment by dachary** [comment]
> I'm not opposed to always producing the full build output

My personal preference is to have full debug output always because I really dislike cryptic error messages. But I also understand why some people find that confusing and prefer terse output when all goes well so they can actually pay attention to the output when something bad happens.

**Comment by heartsucker** [comment]
> My personal preference is to have full debug output always because I really dislike cryptic error messages.

Same. I would rather have full debug output always.

**Comment by redshiftzero** [comment]
This looks great - just needs commits squashed prior to merge

**Pull request merged**